dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -493,6 +504,8 @@ internal class DokkaSymbolVisitor(
                         dri
                     )
                 },
+                contextParameters = if (!contextParametersEnabled) emptyList() else @OptIn(KaExperimentalApi::class) propertySymbol.contextParameters
```

Comment:
Question related to LanguageFeatures: what could be the reason for us to guard calls to `contextParameters`?
AFAIU AA API will just return `emptyList` anyway